tests/test_utils/test_activation_checkpointing.py

```diff
@@ -19,7 +19,7 @@ def forward(x, weight):
 
 
 @pytest.mark.gpu
```

Comment:
@pytest.mark.parametrize(cpu_offload [True, False])
def test_activation_checkpointing(cpu_offload):

Reply:
For we use  `add_seed` in the function. It can not be re-run to test.
Have fixed it and added the pytest.